### DISCUSSION TITLE
Commands extension for Task file-path input?

### DISCUSSION THREAD

**Discussion by pgfearo** [microsoft/vscode-discussions#416] [Extension Development QnA]
This proposed extension would provide convenient file-pick commands to supply file-path input for VS Code tasks.

When a task is invoked with one of the commands the user is prompted with a [Quick Pick list](https://deltaxml.github.io/vscode-xslt-xpath/run-xslt.html#references)  of file-selection options, such as:

- Current file
- Recently used file list - for current task definition 
- Output file-paths list from another previous stage task definition 
- 'Open File' dialog
- 'Save File' dialog
- Clear Recent files list

Would an extension with these commands be useful to other users or extension authors?

**Comment by isidorn** [comment]
I am closing this one since there was not a lot of discussion that it sparked. If there are follow up questions do let me know and we can reopen it. Thank you
